elide-graphql/src/main/java/com/yahoo/elide/graphql/containers/MapEntryContainer.java

```diff
@@ -49,7 +50,7 @@ public Object processFetch(Environment context, PersistentResourceFetcher fetche
                     + "'. Maps only contain fields 'key' and 'value'");
         }
 
-        if (nonEntityDictionary.hasBinding(returnObject.getClass())) {
+        if (nonEntityDictionary.hasBinding(new ClassType(returnObject.getClass()))) {
```

Comment:
We should never call object.getClass directly anywhere in the code.  Always use the getType method in entity dictionary.